models/tanh_rate.h

```diff
@@ -44,22 +44,23 @@ rate model with hyperbolic tangent non-linearity
 Description
 +++++++++++
 
-tanh_rate is an implementation of a nonlinear rate model with input
+``tanh_rate`` is an implementation of a nonlinear rate model with input
 function :math:`input(h) = \tanh(g * (h-\theta))`. It either models a
-rate neuron with input noise (see rate_neuron_ipn), a rate neuron with
-output noise (see rate_neuron_opn) or a rate transformer (see
-rate_transformer_node). Input transformation can either be applied to
+rate neuron with input noise (see ``rate_neuron_ipn``), a rate neuron with
+output noise (see ``rate_neuron_opn``) or a rate transformer (see
+``rate_transformer_node``). Input transformation can either be applied to
 individual inputs or to the sum of all inputs.
 
 The model supports connections to other rate models with either zero or
 non-zero delay, and uses the secondary_event concept introduced with
 the gap-junction framework.
 
 Nonlinear rate neurons can be created by typing
-nest.Create('tanh_rate_ipn') or nest.Create('tanh_rate_opn') for input
+``nest.Create('tanh_rate_ipn')`` or ``nest.Create('tanh_rate_opn')`` for input
```

Comment:
Difficult to have pycodestyle check also the documentation parts, but for consistency:
```suggestion
``nest.Create("tanh_rate_ipn")`` or ``nest.Create("tanh_rate_opn")`` for input
```